### DISCUSSION TITLE
AMP: Atribut dari `amp-custom` Tidak Muncul

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#186] [Q&A] [answered]
``` php
if ($path = File::exist(__DIR__ . '/asset/css/index.min.css')) {
    Asset::style(content($path), 20);
}

```
kang, attribute amp-custom tidak muncul jika css nya dipanggil menggunakan kode diatas

**Comment by taufik-nurrohman** [comment, marked as answer]
Coba prioritas kait `content` ditinggikan. Misal: `Hook::set('content', function($content) {}, 999)`.

**Reply by yuliakbard** [reply]
kang, attribut `amp-custom` hanya muncul pada element `<style></style>` yang memiliki attribut, misalnya saya tambahin id/ class/ amp-boilerplate, elemen `<style></style>` tanpa atribut, tidak memunculkan `amp-custom`

**Reply by taufik-nurrohman** [reply]
Udah fix mas. Regex-nya dibenerin aja, tambahin `?` biar pola atribut jadi opsional:

### Sebelum

~~~ php
'/<style(\s[^>]*?)>/'
~~~

### Sesudah

~~~ php
'/<style(\s[^>]*?)?>/'
~~~

**Reply by yuliakbard** [reply]
kang, bagaimana cara menyembunyikan/ menghilangkan amp-custom, kalau dia punya attribut amp-boilerplate? soalnya ada css wajib halaman AMP yang ada amp-boilerplate
setelah saya mengedit regex nya ada hasil css nya yang seperti ini `<style amp-boilerplate amp-custom>...</style>`

**Reply by taufik-nurrohman** [reply]
Cek update-an.
